### ISSUE TITLE
Support for TF resource AFD Standard/Premium availability  

### ISSUE THREAD

**Issue by nadeenAlshibil** [hashicorp/terraform-provider-azurerm#15205]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Our partners will need to deploy AFD Standard/Premium with Terraform and they are already asking about the resource availability  

### New or Affected Resource(s)

* azurerm_frontdoor

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by sebader** [comment]
Currently WIP. Will be fixed by https://github.com/hashicorp/terraform-provider-azurerm/pull/15140

**Comment by Marcus-James-Adams** [comment]
Duplicate of #13836

**Comment by tombuildsstuff** [comment]
Initial support for this (Profiles/Endpoints) shipped in v3.9.0 - the remaining resources will be split out of https://github.com/hashicorp/terraform-provider-azurerm/pull/16671 then split/refactored as appropriate - however since the initial resources for this have shipped, I'm going to close this issue for the moment.
